### DISCUSSION TITLE
Check rdlc layout connection

### DISCUSSION THREAD

**Discussion by SKeiler** [StefanMaron/BusinessCentral.LinterCop#704] [New Rule]
We have in our projects some problems with the connection of rdlc-layouts to reports.

We have rdlc-Layouts which are not connected to reports (because the rdlc-layout was copied and somebody didn't removed the old one) and also had problems with reports which had a path to a non-existing rdlc-layout. 

So a new rule with a Warning if a rdlc-Layout has no connection to a report and also a report has a connection to a non-existing layout would be really helpful to keep VSC-projects clean.

**Comment by rvanbekkum** [comment]
And apply this rule for any layout type, e.g., RDLC, Word, Excel, External/Custom :)
